packages/wallets/src/features/accounts/screens/ManualDocumentUpload/components/DocumentRuleHint/DocumentRuleHint.scss

```diff
@@ -0,0 +1,15 @@
+.wallets-document-rule-hint {
+    display: flex;
+    width: 156px;
+    padding: 16px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 8px;
+
+    @include mobile {
+        width: 100%;
+        padding: 16px 15.59px;
```

Comment:
Are we not using `rem` anymore? and im not sure where 15.59px coming from.

Reply:
Haa sure2 😅 , i was taking the style from figma and forgot to change it